src/components/ChartRow.js

```diff
@@ -166,7 +167,7 @@ export default class ChartRow extends React.Component {
         const axes = []; // Contains all the yAxis elements used in the render
         const chartList = []; // Contains all the Chart elements used in the render
         // Dimensions
-        const innerHeight = +this.props.height - AXIS_MARGIN * 2;
+        const innerHeight = +this.props.height - this.props.axisMargin * 2;
```

Comment:
Is AXIS_MARGIN unused now?

Reply:
Yes, the `AXIS_MARGIN` constant is completely replaced by the `axisMargin` property.

Reply:
@sartaj10  then I think it can be removed from the top of this file. Other than that let's merge this in.

Reply:
Fixed